general_considerations.md

```diff
@@ -167,11 +200,46 @@ assume the DAG does not have constant difficulty so combining difficulties must
 be done correctly. The solution is to identify the heaviest weight linear path
 from genesis to tip, where the difficulties are summed along the path.
 
+[kp]
+
+We don't need to worry about identifying the heaviest weight linear path. We can
+sum up the difficulties of all the shares between epochs, as I describe in my
+proposal. As long as a share has produced a PoW that is valid for the the block
+the miner is working on, they will be rewarded for it.
+
+In other words, I don't think we can actually ever find a block that will be the
+tip of the chain. If there are 500 miners, there will always be 500 shares in
+flight (give or take). We can't wait for them them converge on a tip. Instead,
+the DAG will converge on a tip naturally, as a share matching bitcoin difficulty
+is found and all miners start producing shares that reference that share.
+
+This goes to the point where shares reference other miner's shares to
+build the DAG.
+
+I think we need to break free from the need to find the heaviest weighted tip. I
+need to be convinced we need this when we have a DAG and a natural convergence
+to a tip every time a share matching bitcoin difficulty is found.
+
+[/kp]
+
+[kptodo]
+
+Drop in a diagram here
+
+[/kptodo]
+
 Finally, we caution that in considering mechanisms to include even smaller
 miners, one must not violate the "progress-free" characteristic of bitcoin. That
 is, one should not sum work from a subset of smaller miners to arrive at a
 higher-work block in the DAG.
 
+[kp]
+
+How can you sum work from smaller miners to arrive at higher-work block? I don't
+understand the reason for this caveat.
+
+[/kp]
+
```

Comment:
Well, this is practically impossible but it comes up a lot in discussion. Progress-free is required for the system to obey the Poisson distribution, among other things, and results in a certain definition of "fairness".